tools/launchdev.sh

```diff
@@ -77,6 +78,8 @@ function init(){
     if [ -z "$ST2_CONF" ]; then
         ST2_CONF=${ST2_REPO}/conf/st2.dev.conf
     fi
+
+    ST2_CONF=$(readlink -f ${ST2_CONF})
```

Comment:
I assume this is to accommodate symlink in your dev env. Will this break existing setup for us?